src/configs.rs

```
@@ -47,3 +61,66 @@ pub const FIXED_SEED: [u8; 32] = [
     89, 15, 245, 34, 234, 120, 17, 218, 167, 20, 216, 9, 59, 62, 123, 217, 29, 137, 138, 115, 62,
     152, 136, 135, 48, 127, 151, 205, 40, 7, 51, 131,
 ];
+
+pub fn set_bloom_fp_rate(
+    config_ctx: &ConfigurationContext,
+    _name: &str,
+    value: &'static ValkeyGILGuard<ValkeyString>,
+) {
+    let fp_rate_str = value.get(config_ctx).to_string_lossy();
+
+    // Try to parse the string value to f64
+    match fp_rate_str.parse::<f64>() {
+        Ok(fp_rate) => {
+            // Validate the range
+            if !(BLOOM_FP_RATE_MIN..=BLOOM_FP_RATE_MAX).contains(&fp_rate) {
+                logging::log_warning(format!(
+                    "FP rate {} out of range [{}, {}]",
+                    fp_rate, BLOOM_FP_RATE_MIN, BLOOM_FP_RATE_MAX
+                ));
+                return;
+            }
+            // If validation passes, update the value
+            BLOOM_FP_RATE.set(config_ctx, value.get(config_ctx).clone());
+            logging::log_notice(format!("Successfully set bloom-fp-rate to {}", fp_rate));
+        }
+        Err(e) => {
+            logging::log_warning(format!(
+                "Invalid value for bloom-fp-rate: {}, error: {}",
+                fp_rate_str, e
+            ));
```

Same here. Let's return an error message